### PR TITLE
Add mission page

### LINKED ISSUE #1289: Host Mission page on website from community repo
- Current location of the page: https://github.com/open-telemetry/community/blob/main/mission-vision-values.md
- Subtask of #833

Subtasks:
- [x] Add page to this repo
- ~Remove page from https://github.com/open-telemetry/community~
- [x] #1809
- [x] https://github.com/open-telemetry/community/pull/1369
- [x] https://github.com/open-telemetry/community/pull/1370
- [ ] #2381

/cc @austinlparker @bhs

### PR DESCRIPTION
**Pull request by svrnm**
- Contributes to #1289

### CODE REVIEW

File: content/en/_index.md [hunk 1]
```diff
@@ -11,10 +11,7 @@ show_banner: true
 <div class="l-primary-buttons mt-5">
 
 - [Learn more](/docs/concepts/)
-- [Mission and vision][]
-
-[mission and vision]:
-  https://github.com/open-telemetry/community/blob/main/mission-vision-values.md#otel-mission-vision-and-values
+- [Mission, Vision & Values](/mission-vision-and-values)
```
Comment (chalin) [inline content/en/_index.md:14]: - Change button text to sentence case
- Use `/mission` as the path (which implies renaming the file, as is noted below)

**Review by chalin — COMMENTED** [review]
Thanks for this @svrnm! See inline comments.

**Comment by chalin** [comment]
Btw, some of the suggested changes are to the original text. Feel free to leave those as is, and if so, I'll make a second pass. Making the updates would be welcome though. Thanks.

File: content/en/_index.md (hunk 1 above)
Comment (svrnm) [reply content/en/_index.md]: fixed in ab513c6

File: content/en/mission-vision-and-values.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
+---
+title: Mission
+menu:
+  main: {weight: 40}
+aliases: [/mission, /vision, /values]
+---
+{{% blocks/lead color="primary" %}}
+# OpenTelemetry &mdash; Mission, Vision and Values
```
[the remark this answers is not in the document]
Comment (svrnm) [reply content/en/mission-vision-and-values.md]: fixed in ab513c6

File: content/en/mission-vision-and-values.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+mission, and as a project we refer to them to orient – and reorient – as we
+chart our path.
+
+## Engineering Values &mdash; the principles that guide our contributions
+
+OpenTelemetry’s mission and vision describe where we want to go.
+OpenTelemetry’s engineering values describe how we want to get there.
+
+OpenTelemetry’s core engineering values are Compatibility, Stability,
+Resiliency, and Performance.
```
[the remark this answers is not in the document]
Comment (svrnm) [reply content/en/mission-vision-and-values.md]: fixed in ab513c6, 86ec953

File: content/en/mission-vision-and-values.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
+---
+title: Mission
+menu:
+  main: {weight: 40}
+aliases: [/mission, /vision, /values]
```
[the remark this answers is not in the document]
Comment (svrnm) [reply content/en/mission-vision-and-values.md]: fixed

**Review by chalin — APPROVED** [review]
Thanks @svrnm!

**Pull request merged**